kotlin/openID4VP/src/commonMain/kotlin/io/mosip/openID4VP/authorizationRequest/clientMetadata/ClientMetadataDraft23.kt

```
@@ -0,0 +1,137 @@
+		element<String>("client_name", isOptional = true)
+		element<String>("logo_uri", isOptional = true)
+		element<Map<String, Map<String, List<String>>>>("vp_formats", isOptional = false)
+		element<String>("authorization_encrypted_response_alg", isOptional = true)
+		element<String>("authorization_encrypted_response_enc", isOptional = true)
+		element<Jwks>("jwks", isOptional = true)
+	}
+
+	override fun deserialize(decoder: Decoder): ClientMetadataDraft23 {
+		val jsonDecoder = try {
+			decoder as JsonDecoder
+		} catch (e: ClassCastException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)